### PR TITLE
Fix Ballot creation bug, closes #163. 

### LINKED ISSUE #163: Ballot create route issue

### PR DESCRIPTION
**Pull request by brianpchsu**
Ballot creation:
screen shot 2015-04-24 at 2 38 56 pm

### CODE REVIEW

File: server/controllers/api/ballots.js [hunk 1]
```diff
@@ -10,40 +10,57 @@ var uuid = require('uuid');
...
-                                                 created_at: ballot.get('created_at') };
-                              res.send(201, response);
-                            });// save ballot
-                      });//fetch ballot
-              });//fetch election
+        .fetch()
+        .then(function(election){
+          if (!election) res.send(400, 'Error: election not found.');
+          //TODO: re-enable check for accepting votes
+          // if (!election.get('accepting_votes')) res.send(403, 'This election is no longer accepting votes.');
```
Comment (ryyann) [inline server/controllers/api/ballots.js]: Might as well re-enable this first before merging

Comment (brianpchsu) [inline server/controllers/api/ballots.js]: Got it. Push in a minute

Comment (brianpchsu) [inline server/controllers/api/ballots.js]: Changed, squashed, and push it back.

**Comment by ryyann** [comment]
:+1:

**Pull request merged**